Assets/_BForBoss/_Core/Scripts/Debug/DebugView/PlayerPrefResetter.cs

```diff
@@ -14,7 +16,9 @@ public class PlayerPrefResetter : DebugView
 
         public PlayerPrefResetter(Rect masterRect) : base(masterRect)
         {
-            _keys = GetConstStringValuesFromStruct<UploadPlayerScoreDataSource.PlayerPrefKey>();
+            _keys = GetConstStringValuesFromStruct<PlayerPrefKeys.InputSettings>();
+            _keys.AddRange(GetConstStringValuesFromStruct<PlayerPrefKeys.ThirdPerson>());
+            _keys.AddRange(GetConstStringValuesFromStruct<PlayerPrefKeys.GameplaySettings>());
```

Comment:
I wonder if it would make sense to move the get const sting values function into PlayerPrefKeys, and just have it output a list for the whole class?

Reply:
yeah that would be better.

Reply:
@unoctium1 do you mind checking out this branch and doing that?

Reply:
Yeah for sure, in that case dw about changing it to static, I can do that as well